e2e/e2eTestRunnerWithCache.php

```diff
@@ -24,6 +24,9 @@
 $output = str_replace(__DIR__, '.', $output);
 
 $expectedDiff = 'expected-output.diff';
+if (isset($argv[1]) && $argv[1] === '--diff') {
```

Comment:
We allow setting which output/diff file should be used so that we can test different outputs in our new end 2 end test